bi_sql_editor/models/bi_sql_view.py

```diff
@@ -652,6 +652,11 @@ def _refresh_materialized_view(self):
                 # Alter name of the action, to display last refresh
                 # datetime of the materialized view
                 sql_view.action_id.name = sql_view._prepare_action_name()
+                #translate action name
+                translations = self.env['ir.translation'].search([('name','=','ir.actions.act_window,name'),
+                                                                 ('res_id','=',sql_view.action_id.id)])
+                for translation in translations:
+                    translation.write({'value': sql_view.action_id.name})
```

Comment:
I think this implementation will disable the ``_prepare_action_name()`` feature. don't you think ?

I think you should better make something like : 

```
for translation in translations:
    sql_view.action_id.with_context(lang=translation.lang).name = sql_view.with_context(lang=translation.lang)._prepare_action_name()
```

or something like that...